runtime/t0rn-parachain/src/xcm_config.rs

```diff
@@ -52,6 +56,32 @@ pub type LocalAssetTransactor = CurrencyAdapter<
     (),
 >;
 
+// The non-reserve fungible transactor type
+// It will use pallet-assets, and the Id will be matched against AsAssetType
+// This is intended to match FOREIGN ASSETS
+pub type ForeignFungiblesTransactor = FungiblesAdapter<
+    // Use this fungibles implementation:
+    Assets,
+    // Use this currency when it is a fungible asset matching the given location or name:
+    ConvertedConcreteAssetId<
+        AssetId,
```

Comment:
Is this assetId correct? it can't be an enum I think. I was having issues getting it to work with enums so had to set it as u32

Reply:
yeah replaced with `common_parachains::AssetId` for transacting which also is u32 https://github.com/paritytech/cumulus/blob/master/parachains/common/src/lib.rs#L65